### DISCUSSION TITLE
Colors are off.... I kind of know why but fixing is another issue

### DISCUSSION THREAD

**Discussion by Rusty-sk** [PlummersSoftwareLLC/NightDriverStrip#92] [Q&A] [answered]
First off I am a web dev that has got into tinkering and I know I heard Dave say something about the new way versus the old way of rendering colors in regards to the WS2812B lights... All I have are about 1100 or so WS-2811 lights (the old way) so while I experiment I would know how I can clean up the color while I look at building a headless rest project to work along side of Dave's great software.

My magenta's are cyan and my blues are bright green.

**Comment by xcalibur839** [comment, marked as answer]
Try adding this to the section you're using in globals.h:
`#define COLOR_ORDER EOrder::RGB`
or if that doesn't work, try changing it to:
`#define COLOR_ORDER EOrder::GRB`

**Reply by Rusty-sk** [reply]
Spectacular. I knew it was a relatively easy fix but since I am focused of other issues I knew about I thought I would ask people much smarter in this area.

TY Xcalibur you slayed the beast.
